### DISCUSSION TITLE
It flickers to change the selection in the candidate list

### DISCUSSION THREAD

**Discussion by HubKing** [fcitx/fcitx5#681] [Q&A] [answered]
If I change the selection in the candidate list of Mozc, very often, the list pop-up briefly disappears and reappears, making the flickering. Please see the screen recording below. You can see that the text beneath the candidate list "ねこねこねこ" keeps briefly appearing. If it is too fast, show the video in slow mode. In my opinion, the list should be constantly on, until the user dismisses the list. 

Is this a bug?

Gnome 43.1, Wayland. Fcitx 5.0.19-1 Official Repositories  (community). Fcitx-mozc 2.26.4632.102 Official Repositories  (community).

[Kooha-2022-12-05-18-53-16.webm](https://user-images.githubusercontent.com/10900489/205607580-562990c3-b0c1-401e-b8a5-4d422778989c.webm)

**Comment by wengxt** [comment, marked as answer]
Try this extension: https://extensions.gnome.org/extension/261/kimpanel/ 

See https://fcitx-im.org/wiki/FAQ#Candidate_window_is_blinking_under_wayland_with_Fcitx_5 for the details reason.

**Reply by HubKing** [reply]
Thanks. But the FAQ page's instruction was not detailed enough, so I had to spend some time to get it working. In short, install that extension, and in Fcitx's config tool -> Addons, check Sow Advanced and then uncheck classic UI and check KDE Input Panel, and log out/in again.
